src/ts_insert.cpp

```diff
@@ -21,18 +21,24 @@
 //' handle <- connect("qdb://127.0.0.1:2836")
 //' ts_insert.double(handle, "timeseries", "column", points)
 // [[Rcpp::export(name = "ts_insert.double")]]
-void _qdb_ts_double_insert(qdb_handle_t handle, const std::string & name, const std::string & column)
+void _qdb_ts_double_insert(
+    qdb_handle_t handle, const std::string & name, const std::string & column)
 {
     if (!handle)
     {
         Rcpp::stop("invalid handle");
     }
-
-    std::vector<qdb_ts_double_point> values{
-        {qdb_timespec_t{1'522'908'428, 123'456'789}, 1.2345} // 2018-04-05T06:07:08.123456789+00:00
+    const std::array<qdb_ts_double_point, 3> values{
```

Comment:
Add a comment that it's only for testing for the moment.
Your future self (and others) will appreciate. :)
Sorry for not having done this before.

Reply:
Done.